src/pygeoprocessing/geoprocessing.py

```diff
@@ -2433,10 +2484,13 @@ def warp_raster(
             in Well Known Text format.
         n_threads (int): optional, if not None this sets the ``N_THREADS``
             option for ``gdal.Warp``.
-        vector_mask_options (dict): optional, if not None, this is a
-            dictionary of options to use an existing vector's geometry to
-            mask out pixels in the target raster that do not overlap the
-            vector's geometry. Keys to this dictionary are:
+        vector_mask_options (dict or str): optional. If None, no masking will
+            be done.  If a string, it must be a path to a raster that is in the
```

Comment:
Small discrepancy here where later on `mask_raster_path` is mentioned as a key in the dict, but here it is suggested to pass directly as a string.

Reply:
Whoops! Yes, this is left over from an earlier draft and patched in 52524630